src/dbcp/transform/eip_infrastructure.py

```diff
@@ -67,114 +89,138 @@ def facilities_transform(raw_fac_df: pd.DataFrame) -> pd.DataFrame:
         pd.DataFrame: transformed copy of the raw facilities dataframe
     """
     fac = raw_fac_df.copy()
-    fac.columns = _format_column_names(fac.columns)
+    fac = _split_json_column(fac, col="xata")
+    fac.columns = _format_column_names(fac)
+    # Drop facilities that aren't published by EIP
+    fac = fac.loc[fac["published"]]
```

Comment:
What does it mean to be published by EIP? Why do we only want the facilities published by EIP?